docs/_templates/no-frame.njk

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
```

Comment:
This template/layout removes the HelixUI app frame so that I can position elements flush against the edges of the viewport.